### ISSUE TITLE
Gitcoin: 9) Initiate Withdrawal Process from the Layer 2 back to Layer 1

### ISSUE THREAD

**Issue by matt-quinn** [nervosnetwork/grants#10]
This task is capped at maximum 175 submissions; so be in the first 175 valid submissions to get your payout!

# 9. Initiate a Withdrawal From Layer 2 Back to Layer 1

Withdrawing an asset from [Layer 2](https://github.com/Kuzirashi/gw-gitcoin-instruction/blob/master/src/conceptual-explainers/structure.md#layer-1--layer-2) back to [Layer 1](https://github.com/Kuzirashi/gw-gitcoin-instruction/blob/master/src/conceptual-explainers/structure.md#layer-1--layer-2) is another step in the roundtrip process of interoperability. [Godwoken](https://github.com/Kuzirashi/gw-gitcoin-instruction/blob/master/src/conceptual-explainers/frameworks.md#godwoken) is a Layer 2 that is a natural extension of Layer 1, so it has deep integration which allows it to function seamlessly using a built-in withdrawal mechanism.

Deposits from Layer 1 to Layer 2 are quick. This is because Layer 1 has stronger security than Layer 2. When a deposit is made to Layer 2, it can reliably trust Layer 1 after waiting for only a few confirmations. However, a withdrawal is the opposite. When a withdrawal is made to Layer 1, it must rely on Layer 2, which inherently has lower security. For this reason, the number of confirmations required is much higher in order to guarantee that the security remains reliable. It takes approximately five days from the start of the withdrawal process to the point where funds are actually released on Layer 1.

This long withdrawal duration may seem exceptionally long, but it is a necessary step at this stage. Godwoken uses an optimistic rollup architecture that only allows a single honest node in the network. This provides an exceptionally secure basis for Layer 2, but the tradeoff is that it requires a "challenge period" when exiting from Layer 2. This is a period where Layer 2 network operators have time to check and raise issues with any potentially malicious transactions and rollback if necessary. The safety of the network is the first concern. Network parameters may be changed in the future, but only after it has been proven to be safe. It is also possible that other solutions will be created in the future to allow for faster exits from Layer 2.

## Task Instructions

> Note: Before starting the tasks, it is recommended that you review the [Task Submission](#task-submission) section so you know what materials you will need to provide to judges to review your task submission.

In this task you will use Godwoken Examples tool package to execute the `withdraw` command using `account-cli.js`. This is the first part of the withdrawal process, and the funds must still undergo a five day waiting period before they are unlocked and released. To complete this task, you must issue the `withdraw` command to start the process. You will still have to wait five days before unlocking to complete the process, but this second part has been split into the next task.

### Prerequisites

Before you begin on this task you must complete the [first task](https://github.com/Kuzirashi/gw-gitcoin-instruction/tree/master/src/tasks/1.create.godwoken.account.md) to create a Godwoken account on the EVM Layer 2 Testnet. You will also need the private key and Testnet address from the first task in order to proceed. If you have not completed it, please do so now.

You can find setup instructions for Godwoken Examples Tools Package [here](https://github.com/Kuzirashi/gw-gitcoin-instruction/blob/master/src/component-tutorials/3.setup.and.use.account.cli.md#setup-the-godwoken-examples-tools-package).

### 1. Execute the Withdraw Command

Using the Godwoken Examples tool package, execute the `withdraw` command using the command below, replacing values as needed.

```sh
cd godwoken-examples
node ./packages/tools/lib/account-cli.js withdraw -r <INSERT_TESTNET_RPC_URL> -p <YOUR_ETHEREUM_PRIVATE_KEY> -o <YOUR_LAYER_1_ADDRESS> -c <SHANNONS_AMOUNT>
```

1. `<INSERT_TESTNET_RPC_URL>` - The RPC URL of the Testnet CKB node you will use. This should be your local address if you are using a local node, or it can be a public address if you're using a public node.\
Local Testnet CKB Node URL: `http://127.0.0.1:8114`\
Public Testnet CKB Node URL: `http://3.235.223.161:18114`
2. `<YOUR_ETHEREUM_PRIVATE_KEY>` - This is your Ethereum private key for the Layer 2 account that contains the CKBytes which were deposited previously. **Always make sure your private key is prefixed with "0x".**
3. `<YOUR_LAYER_1_ADDRESS>` - The Nervos CKB Layer 1 address for your account on Layer 1. This will be used to receive funds from Layer 2. If you used `ckb-cli` to generate your accounts, you can view them using the `account list` command.
4. `<SHANNONS_AMOUNT>` - The amount of CKBytes to transfer, written in Shannons. A Shannon is a unit of measurement in Nervos that is the equivalent of a Satoshi in Bitcoin. We recommend depositing 400 CKBytes, which is the minimum number required by Godwoken. There are 100,000,000 Shannons in a CKByte, and this value must be expressed in Shannons. ie: 400 * 100,000,000 = 40,000,000,000

Example final command with all the values filled in:

```sh
cd godwoken-examples
node ./packages/tools/lib/account-cli.js withdraw -r http://127.0.0.1:8114 -p 0xd9066ff9f753a1898709b568119055660a77d9aae4d7a4ad677b8fb3d2a571e5 -o ckt1qyq9u5vzgtklnqrr6cevra7w2utrsxmjgefs72sfju -c 40000000000
```

<details>
<summary>Example Output (click to expand)</summary>
  
```txt
LUMOS_CONFIG_NAME: AGGRON4
owner lock hash: 0x5c7253696786b9eddd34e4f6b6e478ec5742bd36569ec60c1d0487480ba4f9e3
eth address: 0xd173313a51f8fc37bcf67569b463abd89d81844f
l2 lock hash: 0x8016dcd1af7c8cceda53e4d2d2cd4e2924e245b629e0f81f3f64969787b2b049
--- godwoken withdraw ---
nonce: 92
rollupTypeHash: 0x4cc2e6526204ae6a2e8fcf12f7ad472f41a1606d5b9624beebd215d780809f6a
withdrawalRequest: {
  raw: {
    nonce: '0x5c',
    capacity: '0x9502f9000',
    amount: '0x0',
    sudt_script_hash: '0x0000000000000000000000000000000000000000000000000000000000000000',
    account_script_hash: '0x8016dcd1af7c8cceda53e4d2d2cd4e2924e245b629e0f81f3f64969787b2b049',
    sell_amount: '0x0',
    sell_capacity: '0x2540be400',
    owner_lock_hash: '0x5c7253696786b9eddd34e4f6b6e478ec5742bd36569ec60c1d0487480ba4f9e3',
    payment_lock_hash: '0x0000000000000000000000000000000000000000000000000000000000000000',
    fee: { sudt_id: 1, amount: 0n }
  },
  signature: '0xd9f5bf8fa5ff21fd8d43b5cd09122938ef9c84825f633d645432bd61763253f323ab9579328a5f4a4b09d8247fae1b4d1788d8c956d79cc0769e507d8643820101'
}
result: null
--- godwoken withdraw finished ---
Your account id: 13
ckb balance in godwoken is: 399999661932
Success! Withdrawal request sent. You need to wait now for challenge period duration to unlock the funds.
```

</details>

If you see *"Success! Withdrawal request sent. You need to wait now for challenge period duration to unlock the funds."* message at the end it means you have successfully initiated a withdrawal process. Congratulations, you have completed the task! 👏

Withdrawing from Nervos Layer 2 is a two step process, first the withdrawal is initiated with the `withdrawl` command, and then it is later unlocked with the `unlock` command after the five day challenge period. However, executing the `unlock` command is part of the next task, and is not needed for this task.

## Task Submission

To complete the tasks, add the following materials to a document on your Github and submit for review by the judges (include the link in your Gitcoin submission):

1. A screenshot of the console output immediately after running the `withdraw` command.
2. The Ethereum address that you've used for your Layer 2 account (in text format).
3. The Nervos Layer 1 address that you passed to `withdraw` command (in text format).

## Helpful Links

- [Introduction](https://github.com/Kuzirashi/gw-gitcoin-instruction/blob/master/src/introduction/introduction.md)
- [Task Setup and Requirements](https://github.com/Kuzirashi/gw-gitcoin-instruction/tree/master/src/task-setup-and-requirements)
- [Discord](https://discord.com/invite/AqGTUE9)
- [Broaden the Spectrum](https://gitcoin.co/hackathon/nervos/onboard)
- [Bounties List](https://gitcoin.co/hackathon/nervos/)
- [Content Directory](https://github.com/Kuzirashi/gw-gitcoin-instruction)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work has been started__.


These users each claimed they can complete the work by 2 weeks, 4 days from now.
Please review their action plans below:


**1) [timfaner](https://gitcoin.co/timfaner) has started work.**

Task9 submission
**2) [sidduhere](https://gitcoin.co/sidduhere) has started work.**

Withdrawal Process
**3) [soptq](https://gitcoin.co/soptq) has started work.**

I Will Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**4) [vinhbhn](https://gitcoin.co/vinhbhn) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**5) [antonyip](https://gitcoin.co/antonyip) has started work.**

Withdrawal ckb
**6) [rickmort](https://gitcoin.co/rickmort) has started work.**

I'll initiate the withdraw process from the layer 2 back to layer 1
**7) [jinusean](https://gitcoin.co/jinusean) has started work.**

Gitcoin: 9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**8) [rezahsnz](https://gitcoin.co/rezahsnz) has started work.**

nervos t9
**9) [assafom](https://gitcoin.co/assafom) has started work.**

Step 9.
**10) [l-kh](https://gitcoin.co/l-kh) has started work.**

this is my submission for Nervos Hackathon Gitcoin9
**11) [jpeterd](https://gitcoin.co/jpeterd) has started work.**

Withdraw from Layer 2 back to Layer 1
**12) [missgeds](https://gitcoin.co/missgeds) has started work.**

Task submission
**13) [hsamndo](https://gitcoin.co/hsamndo) has started work.**

I will learn and apply how to initiate. withdrawal process from the layer 2 back to layer
**14) [7h2x5e](https://gitcoin.co/7h2x5e) has started work.**

Task 9 submission
**15) [orangemio](https://gitcoin.co/orangemio) has started work.**

https://github.com/orangemio/nervos-grants/tree/main/task9
**16) [domix14](https://gitcoin.co/domix14) has started work.**

nervos task 9
**17) [andithemudkip](https://gitcoin.co/andithemudkip) has started work.**

Task 9
**18) [maxx6262](https://gitcoin.co/maxx6262) has started work.**

Gitcoin: 9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**19) [johnalgo](https://gitcoin.co/johnalgo) has started work.**

Nervos
**20) [happylolonly](https://gitcoin.co/happylolonly) has started work.**

https://github.com/happylolonly/nervos-hackathon/tree/main/task-9
**21) [x777](https://gitcoin.co/x777) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**22) [huutrung](https://gitcoin.co/huutrung) has started work.**

Task9
**23) [pgonday](https://gitcoin.co/pgonday) has started work.**

Gitcoin hackaton
**24) [rbchi](https://gitcoin.co/rbchi) has started work.**

sss
**25) [alexchun1011](https://gitcoin.co/alexchun1011) has started work.**

Task09
**26) [dex68](https://gitcoin.co/dex68) has started work.**

ok
**27) [juzkiddin](https://gitcoin.co/juzkiddin) has started work.**

This is task 9 of Nervos Hackathon
**28) [muhdmud](https://gitcoin.co/muhdmud) has started work.**

nervos network
**29) [luisantoniocrag](https://gitcoin.co/luisantoniocrag) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**30) [yasinkirazci](https://gitcoin.co/yasinkirazci) has started work.**

very very good projes
**31) [francov99](https://gitcoin.co/francov99) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**32) [anhnt4288](https://gitcoin.co/anhnt4288) has started work.**

Nervous hackathon gitcoin
**33) [ttnguyendev](https://gitcoin.co/ttnguyendev) has started work.**

A project for hackathon
**34) [raiden1411](https://gitcoin.co/raiden1411) has started work.**

Nervos 9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**35) [maxencealvarez](https://gitcoin.co/maxencealvarez) has started work.**

Task 9
**36) [tharunrai14](https://gitcoin.co/tharunrai14) has started work.**

Nervos hacakthon task 09
**37) [wangjianyeart](https://gitcoin.co/wangjianyeart) has started work.**

good
**38) [mirrormirage0](https://gitcoin.co/mirrormirage0) has started work.**

Withdraw tokens from L2 to L1
**39) [ririen](https://gitcoin.co/ririen) has started work.**

This project documents and focuses on starting the layer 2 initiate withdrawal process.
**40) [s1119858711](https://gitcoin.co/s1119858711) has started work.**

9
**41) [jeasterman](https://gitcoin.co/jeasterman) has started work.**

Task 9
**42) [mrjacobsullivan](https://gitcoin.co/mrjacobsullivan) has started work.**

Nervos - Broaden the Spectrum
**43) [gapro8](https://gitcoin.co/gapro8) has started work.**

task 9
**44) [nhaga](https://gitcoin.co/nhaga) has started work.**

Nervos
**45) [luiscrag](https://gitcoin.co/luiscrag) has started work.**

I'll learn how to withdraw from the layer 2 to the layer 1.
**46) [lofravaz](https://gitcoin.co/lofravaz) has started work.**

Initiate Withdrawal Process from the Layer 2 back to Layer 1
**47) [andermarce](https://gitcoin.co/andermarce) has started work.**

Task9
**48) [cesheep](https://gitcoin.co/cesheep) has started work.**

Nervos Step 9
**49) [walkertraylor](https://gitcoin.co/walkertraylor) has started work.**

Nervos Hackathon Gitcoin: 9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**50) [pfed-prog](https://gitcoin.co/pfed-prog) has started work.**

project 9
**51) [baoanh1310](https://gitcoin.co/baoanh1310) has started work.**

Initiate withdrawal process from Layer 2
**52) [kidneyweakx](https://gitcoin.co/kidneyweakx) has started work.**

Withdrawling an asset from Layer 2 back to Layer 1.
**53) [ubinix-warun](https://gitcoin.co/ubinix-warun) has started work.**

1. A screenshot of the console output immediately after running the withdraw command.
2. The Ethereum address that you've used for your Layer 2 account (in text format).
3. The Nervos Layer 1 address that you passed to withdraw command (in text format).
**54) [duylinh196tb](https://gitcoin.co/duylinh196tb) has started work.**

Nervos gitcoin task 9
**55) [lehoi2195](https://gitcoin.co/lehoi2195) has started work.**

Initiate a Withdrawal From Layer 2 Back to Layer 1
**56) [nishuzumi](https://gitcoin.co/nishuzumi) has started work.**

nervos
**57) [quocson95](https://gitcoin.co/quocson95) has started work.**

nervos-task 9
**58) [sondq](https://gitcoin.co/sondq) has started work.**

nervos 9
**59) [quocson100](https://gitcoin.co/quocson100) has started work.**

task 9
**60) [bayou020](https://gitcoin.co/bayou020) has started work.**

Task9
**61) [selimkafa](https://gitcoin.co/selimkafa) has started work.**

nervos9
**62) [dragondev1906](https://gitcoin.co/dragondev1906) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**63) [niedhui](https://gitcoin.co/niedhui) has started work.**

Task 9
**64) [dikky88](https://gitcoin.co/dikky88) has started work.**

ckb dikky task9
**65) [jcervante](https://gitcoin.co/jcervante) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**66) [celoaken](https://gitcoin.co/celoaken) has started work.**

Initiate withdrawal
**67) [simiku](https://gitcoin.co/simiku) has started work.**

Gitcoin: 9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**68) [pulth](https://gitcoin.co/pulth) has started work.**

NervosHackathon Task9 Submission
**69) [armujahid](https://gitcoin.co/armujahid) has started work.**

L2 to L1 withdrawal
**70) [hodlrtodlrfarmr](https://gitcoin.co/hodlrtodlrfarmr) has started work.**

Gitcoin: 9 Submission
**71) [phungnm](https://gitcoin.co/phungnm) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**72) [jackieyewang](https://gitcoin.co/jackieyewang) has started work.**

https://github.com/Jackieyewang/Nervos-gitcoin/tree/main/task9
**73) [ody123](https://gitcoin.co/ody123) has started work.**

task9
**74) [superheavytoken](https://gitcoin.co/superheavytoken) has started work.**

Stage 10 Hackathon Nervos
**75) [ben-razor](https://gitcoin.co/ben-razor) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**76) [gentlebutapril](https://gitcoin.co/gentlebutapril) has started work.**

task9
**77) [terryboy](https://gitcoin.co/terryboy) has started work.**

task9
**78) [scentlxss](https://gitcoin.co/scentlxss) has started work.**

nervos task9
**79) [deepansharya1111](https://gitcoin.co/deepansharya1111) has started work.**

This is my Submission for the task: "Gitcoin: 9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1"
**80) [gabrycina](https://gitcoin.co/gabrycina) has started work.**

Nervos Hackaton Task 9
**81) [theyashmhatre](https://gitcoin.co/theyashmhatre) has started work.**

Task 9
**82) [u1dxpool](https://gitcoin.co/u1dxpool) has started work.**

nervos_gitcoin
**83) [ufrok1](https://gitcoin.co/ufrok1) has started work.**

NervosGitCoin
**84) [u2rust](https://gitcoin.co/u2rust) has started work.**

NervosGitcoin
**85) [w5pand](https://gitcoin.co/w5pand) has started work.**

NervosMoon
**86) [knnlrts](https://gitcoin.co/knnlrts) has started work.**

Nervos hackaton: task 9
**87) [rzbck](https://gitcoin.co/rzbck) has started work.**

Gitcoin: 9- Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**88) [elfelf2](https://gitcoin.co/elfelf2) has started work.**

task 9
**89) [stone-rocket](https://gitcoin.co/stone-rocket) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**90) [superburst](https://gitcoin.co/superburst) has started work.**

task9
**91) [meathewww](https://gitcoin.co/meathewww) has started work.**

gticoin task 9
**92) [ircrp](https://gitcoin.co/ircrp) has started work.**

Start the withdrawal from Layer 2 back to Layer 1
**93) [demoncash08](https://gitcoin.co/demoncash08) has started work.**

Withdrawal from layer 2
**94) [sabaiprimo](https://gitcoin.co/sabaiprimo) has started work.**

Project for Nervos Hackathon task 9
**95) [andrewpsp](https://gitcoin.co/andrewpsp) has started work.**

Welcome to the Yard
**96) [ysongh](https://gitcoin.co/ysongh) has started work.**

I will initiate Withdrawal Process from the Layer 2 back to Layer 1
**97) [berkayermis](https://gitcoin.co/berkayermis) has started work.**

Submissions of Task 9
**98) [homerosland](https://gitcoin.co/homerosland) has started work.**

task9
**99) [mistakeone](https://gitcoin.co/mistakeone) has started work.**

task 9
**100) [ayush20](https://gitcoin.co/ayush20) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**101) [preet-aulokh](https://gitcoin.co/preet-aulokh) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**102) [liushooter](https://gitcoin.co/liushooter) has started work.**

ckb
**103) [awaissaeeed](https://gitcoin.co/awaissaeeed) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**104) [asfman](https://gitcoin.co/asfman) has started work.**

task 9
**105) [mashmzc](https://gitcoin.co/mashmzc) has started work.**

Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**106) [nicky-ru](https://gitcoin.co/nicky-ru) has started work.**

In this task I will withdraw some funds from layer 2 to layer 1
**107) [hhuuaa](https://gitcoin.co/hhuuaa) has started work.**

d
**108) [huazhouwang](https://gitcoin.co/huazhouwang) has started work.**

d
**109) [puppetmaster66](https://gitcoin.co/puppetmaster66) has started work.**

Task 9
**110) [aliabdullah54](https://gitcoin.co/aliabdullah54) has started work.**

task 09
**111) [howinoc](https://gitcoin.co/howinoc) has started work.**

gitcoin9
**112) [drugurares](https://gitcoin.co/drugurares) has started work.**

Gitcoin Nervos Hackathon
**113) [vivensy](https://gitcoin.co/vivensy) has started work.**

task9
**114) [xinbadev](https://gitcoin.co/xinbadev) has started work.**

nervos-gitcoin
**115) [deryyy](https://gitcoin.co/deryyy) has started work.**

tash 9 done
**116) [voldown](https://gitcoin.co/voldown) has started work.**

task 9 submission.
**117) [fffrrreeegithub](https://gitcoin.co/fffrrreeegithub) has started work.**

Nervos Hackathon Lvl9
**118) [bcvsv](https://gitcoin.co/bcvsv) has started work.**

Nervos(9)
**119) [jiddukrish](https://gitcoin.co/jiddukrish) has started work.**

task9
**120) [chaitanyasjoshi](https://gitcoin.co/chaitanyasjoshi) has started work.**

Initiated Withdrawal Process From The Layer 2 Back To Layer 1
**121) [techmeupnl](https://gitcoin.co/techmeupnl) has started work.**

Hackathon
**122) [thanhnhann](https://gitcoin.co/thanhnhann) has started work.**

Done Nervos-task9
**123) [el-tumero](https://gitcoin.co/el-tumero) has started work.**

Ninth encounter with Nervos.
**124) [fabioktldaf](https://gitcoin.co/fabioktldaf) has started work.**

9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**125) [stgllr](https://gitcoin.co/stgllr) has started work.**

nervos-gitcoin
**126) [da3dalus88](https://gitcoin.co/da3dalus88) has started work.**

NN Hackathon
**127) [arnaudatcomet](https://gitcoin.co/arnaudatcomet) has started work.**

HKT Nervos #9
**128) [schrubitteflau](https://gitcoin.co/schrubitteflau) has started work.**

Hackathon: Nervos - Broaden the Spectrum
Gitcoin: 9) Initiate Withdrawal Process from the Layer 2 back to Layer 1
**129) [larricane](https://gitcoin.co/larricane) has started work.**

Gitcoin: 9) Initiate Withdrawal Process From The Layer 2 Back To Layer 1
**130) [dziobakwszafie](https://gitcoin.co/dziobakwszafie) has started work.**

Nervos
**131) [oceansxq](https://gitcoin.co/oceansxq) has started work.**

nervos-gitcoin-9
**132) [wizardlog](https://gitcoin.co/wizardlog) has started work.**

withdraw
**133) [cito-lito](https://gitcoin.co/cito-lito) has started work.**

nervos task 9
**134) [flippinroo2](https://gitcoin.co/flippinroo2) has started work.**

This repository is to house submissions for the Nervos hackathon on GitCoin.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/nervosnetwork/grants/10/100026216).

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work for 750.0 CKB (7.57 USD @ $0.01/CKB) has been submitted by__:

 1. [@timfaner](https://gitcoin.co/timfaner)
 2. [@sidduhere](https://gitcoin.co/sidduhere)
 3. [@soptq](https://gitcoin.co/soptq)
 4. [@vinhbhn](https://gitcoin.co/vinhbhn)
 5. [@antonyip](https://gitcoin.co/antonyip)
 6. [@rickmort](https://gitcoin.co/rickmort)
 7. [@jinusean](https://gitcoin.co/jinusean)
 8. [@rezahsnz](https://gitcoin.co/rezahsnz)
 9. [@assafom](https://gitcoin.co/assafom)
 10. [@l-kh](https://gitcoin.co/l-kh)
 11. [@jpeterd](https://gitcoin.co/jpeterd)
 12. [@missgeds](https://gitcoin.co/missgeds)
 13. [@hsamndo](https://gitcoin.co/hsamndo)
 14. [@7h2x5e](https://gitcoin.co/7h2x5e)
 15. [@orangemio](https://gitcoin.co/orangemio)
 16. [@domix14](https://gitcoin.co/domix14)
 17. [@taurenshaman](https://gitcoin.co/taurenshaman)
 18. [@xcshuan](https://gitcoin.co/xcshuan)
 19. [@andithemudkip](https://gitcoin.co/andithemudkip)
 20. [@cryptowingnut](https://gitcoin.co/cryptowingnut)
 21. [@schen1](https://gitcoin.co/schen1)
 22. [@tungtobe](https://gitcoin.co/tungtobe)
 23. [@johnalgo](https://gitcoin.co/johnalgo)
 24. [@criusxx](https://gitcoin.co/criusxx)
 25. [@happylolonly](https://gitcoin.co/happylolonly)
 26. [@ndduc-fpt](https://gitcoin.co/ndduc-fpt)
 27. [@x777](https://gitcoin.co/x777)
 28. [@huutrung](https://gitcoin.co/huutrung)
 29. [@pgonday](https://gitcoin.co/pgonday)
 30. [@alexchun1011](https://gitcoin.co/alexchun1011)
 31. [@dex68](https://gitcoin.co/dex68)
 32. [@juzkiddin](https://gitcoin.co/juzkiddin)
 33. [@muhdmud](https://gitcoin.co/muhdmud)
 34. [@luisantoniocrag](https://gitcoin.co/luisantoniocrag)
 35. [@yasinkirazci](https://gitcoin.co/yasinkirazci)
 36. [@anhnt4288](https://gitcoin.co/anhnt4288)
 37. [@sidharthpunathil](https://gitcoin.co/sidharthpunathil)
 38. [@francov99](https://gitcoin.co/francov99)
 39. [@ttnguyendev](https://gitcoin.co/ttnguyendev)
 40. [@raiden1411](https://gitcoin.co/raiden1411)
 41. [@maxencealvarez](https://gitcoin.co/maxencealvarez)
 42. [@asinghvishen](https://gitcoin.co/asinghvishen)
 43. [@wangjianyeart](https://gitcoin.co/wangjianyeart)
 44. [@tharunrai14](https://gitcoin.co/tharunrai14)
 45. [@maxx6262](https://gitcoin.co/maxx6262)
 46. [@ririen](https://gitcoin.co/ririen)
 47. [@mirrormirage0](https://gitcoin.co/mirrormirage0)
 48. [@s1119858711](https://gitcoin.co/s1119858711)
 49. [@jeasterman](https://gitcoin.co/jeasterman)
 50. [@mrjacobsullivan](https://gitcoin.co/mrjacobsullivan)
 51. [@gapro8](https://gitcoin.co/gapro8)
 52. [@nhaga](https://gitcoin.co/nhaga)
 53. [@luiscrag](https://gitcoin.co/luiscrag)
 54. [@ronalddas](https://gitcoin.co/ronalddas)
 55. [@lofravaz](https://gitcoin.co/lofravaz)
 56. [@andermarce](https://gitcoin.co/andermarce)
 57. [@nervos2021](https://gitcoin.co/nervos2021)
 58. [@orhanors](https://gitcoin.co/orhanors)
 59. [@cesheep](https://gitcoin.co/cesheep)
 60. [@walkertraylor](https://gitcoin.co/walkertraylor)
 61. [@pfed-prog](https://gitcoin.co/pfed-prog)
 62. [@long-blade](https://gitcoin.co/long-blade)
 63. [@kidneyweakx](https://gitcoin.co/kidneyweakx)
 64. [@baoanh1310](https://gitcoin.co/baoanh1310)
 65. [@duylinh196tb](https://gitcoin.co/duylinh196tb)
 66. [@lehoi2195](https://gitcoin.co/lehoi2195)
 67. [@nishuzumi](https://gitcoin.co/nishuzumi)
 68. [@quocson95](https://gitcoin.co/quocson95)
 69. [@tech-engine](https://gitcoin.co/tech-engine)
 70. [@ubinix-warun](https://gitcoin.co/ubinix-warun)
 71. [@haxyz](https://gitcoin.co/haxyz)
 72. [@sondq](https://gitcoin.co/sondq)
 73. [@quocson100](https://gitcoin.co/quocson100)
 74. [@bayou020](https://gitcoin.co/bayou020)
 75. [@selimkafa](https://gitcoin.co/selimkafa)
 76. [@dragondev1906](https://gitcoin.co/dragondev1906)
 77. [@niedhui](https://gitcoin.co/niedhui)
 78. [@dikky88](https://gitcoin.co/dikky88)
 79. [@chiragbadhe](https://gitcoin.co/chiragbadhe)
 80. [@jcervante](https://gitcoin.co/jcervante)
 81. [@celoaken](https://gitcoin.co/celoaken)
 82. [@simiku](https://gitcoin.co/simiku)
 83. [@armujahid](https://gitcoin.co/armujahid)
 84. [@pulth](https://gitcoin.co/pulth)
 85. [@hodlrtodlrfarmr](https://gitcoin.co/hodlrtodlrfarmr)
 86. [@phungnm](https://gitcoin.co/phungnm)
 87. [@jackieyewang](https://gitcoin.co/jackieyewang)
 88. [@ody123](https://gitcoin.co/ody123)
 89. [@postman56](https://gitcoin.co/postman56)
 90. [@andres-andretti](https://gitcoin.co/andres-andretti)
 91. [@superheavytoken](https://gitcoin.co/superheavytoken)
 92. [@ben-razor](https://gitcoin.co/ben-razor)
 93. [@gentlebutapril](https://gitcoin.co/gentlebutapril)
 94. [@terryboy](https://gitcoin.co/terryboy)
 95. [@scentlxss](https://gitcoin.co/scentlxss)
 96. [@deepansharya1111](https://gitcoin.co/deepansharya1111)
 97. [@gabrycina](https://gitcoin.co/gabrycina)
 98. [@theyashmhatre](https://gitcoin.co/theyashmhatre)
 99. [@u1dxpool](https://gitcoin.co/u1dxpool)
 100. [@ufrok1](https://gitcoin.co/ufrok1)
 101. [@u2rust](https://gitcoin.co/u2rust)
 102. [@w5pand](https://gitcoin.co/w5pand)
 103. [@caneryy](https://gitcoin.co/caneryy)
 104. [@knnlrts](https://gitcoin.co/knnlrts)
 105. [@rzbck](https://gitcoin.co/rzbck)
 106. [@elfelf2](https://gitcoin.co/elfelf2)
 107. [@stone-rocket](https://gitcoin.co/stone-rocket)
 108. [@superburst](https://gitcoin.co/superburst)
 109. [@meathewww](https://gitcoin.co/meathewww)
 110. [@demoncash08](https://gitcoin.co/demoncash08)
 111. [@sabaiprimo](https://gitcoin.co/sabaiprimo)
 112. [@andrewpsp](https://gitcoin.co/andrewpsp)
 113. [@ircrp](https://gitcoin.co/ircrp)
 114. [@ysongh](https://gitcoin.co/ysongh)
 115. [@berkayermis](https://gitcoin.co/berkayermis)
 116. [@homerosland](https://gitcoin.co/homerosland)
 117. [@mistakeone](https://gitcoin.co/mistakeone)
 118. [@ayush20](https://gitcoin.co/ayush20)
 119. [@preet-aulokh](https://gitcoin.co/preet-aulokh)
 120. [@awaissaeeed](https://gitcoin.co/awaissaeeed)
 121. [@asfman](https://gitcoin.co/asfman)
 122. [@mashmzc](https://gitcoin.co/mashmzc)
 123. [@nicky-ru](https://gitcoin.co/nicky-ru)
 124. [@liushooter](https://gitcoin.co/liushooter)
 125. [@hhuuaa](https://gitcoin.co/hhuuaa)
 126. [@huazhouwang](https://gitcoin.co/huazhouwang)
 127. [@puppetmaster66](https://gitcoin.co/puppetmaster66)
 128. [@aliabdullah54](https://gitcoin.co/aliabdullah54)
 129. [@mmcel](https://gitcoin.co/mmcel)
 130. [@icaroo](https://gitcoin.co/icaroo)
 131. [@bulentbozdag](https://gitcoin.co/bulentbozdag)
 132. [@howinoc](https://gitcoin.co/howinoc)
 133. [@drugurares](https://gitcoin.co/drugurares)
 134. [@vivensy](https://gitcoin.co/vivensy)
 135. [@xinbadev](https://gitcoin.co/xinbadev)
 136. [@deryyy](https://gitcoin.co/deryyy)
 137. [@voldown](https://gitcoin.co/voldown)
 138. [@bcvsv](https://gitcoin.co/bcvsv)
 139. [@jiddukrish](https://gitcoin.co/jiddukrish)
 140. [@chaitanyasjoshi](https://gitcoin.co/chaitanyasjoshi)
 141. [@techmeupnl](https://gitcoin.co/techmeupnl)
 142. [@alanfreud](https://gitcoin.co/alanfreud)
 143. [@thanhnhann](https://gitcoin.co/thanhnhann)
 144. [@el-tumero](https://gitcoin.co/el-tumero)
 145. [@amrosaeed](https://gitcoin.co/amrosaeed)
 146. [@fabioktldaf](https://gitcoin.co/fabioktldaf)
 147. [@stgllr](https://gitcoin.co/stgllr)
 148. [@taylorkhalifa](https://gitcoin.co/taylorkhalifa)
 149. [@da3dalus88](https://gitcoin.co/da3dalus88)
 150. [@arnaudatcomet](https://gitcoin.co/arnaudatcomet)
 151. [@larricane](https://gitcoin.co/larricane)
 152. [@dziobakwszafie](https://gitcoin.co/dziobakwszafie)
 153. [@schrubitteflau](https://gitcoin.co/schrubitteflau)
 154. [@oceansxq](https://gitcoin.co/oceansxq)
 155. [@wizardlog](https://gitcoin.co/wizardlog)
 156. [@cito-lito](https://gitcoin.co/cito-lito)



@mattquinn-nervos please take a look at the submitted work:
* [PR](https://github.com/cito-lito/nervos_hackathon/tree/main/task_09) by @cito-lito
* [PR](https://github.com/wizardlog/gitcoin-nervos-hackacton/tree/master/gitcoin-9) by @wizardlog
* [PR](https://github.com/oceanSxq/nervos-gitcoin/tree/main/ckb-task-9) by @oceansxq
* [PR](https://github.com/Schrubitteflau/Nervos-Hackaton/tree/master/9-Withdraw-Layer2-Layer1) by @schrubitteflau
* [PR](https://github.com/dziobakwszafie/nervos-spectrum/tree/master/Task9) by @dziobakwszafie
* [PR](https://github.com/larricane/nervos-gitcoin/tree/main/task6) by @larricane
* [PR](https://github.com/arnaudatcomet/nervos/tree/main/task-9) by @arnaudatcomet
* [PR](https://github.com/da3dalus88/nervosHackathon/blob/main/tasks/task9/task9.md) by @da3dalus88
* [PR](https://github.com/taylorkhalifa/nervos-gitcoin-hackathon/tree/master/gitcoin-9)Withdraw) by @taylorkhalifa
* [PR](https://github.com/stgllr/nervos-gitcoin/tree/main/Assets) by @stgllr
* [PR](https://github.com/fabioktldaf/nervosnetwork/tree/main/09) by @fabioktldaf
* [PR](https://github.com/amrosaeed/Nervos-Hackathon/tree/master/Task%209) by @amrosaeed
* [PR](https://github.com/ThanhNhann/nervos/tree/main/task9) by @thanhnhann
* [PR](https://github.com/alanfreud/Nervos-gitcoin-projects/tree/master/gitcoin-9) by @alanfreud
* [PR](https://github.com/stevenhva/NervosNetworkHackathon/blob/main/Task_9.md) by @techmeupnl
* [PR](https://github.com/chaitanyasjoshi/nervos-hackathon/tree/main/Task%209) by @chaitanyasjoshi
* [PR](https://github.com/jiddukrish/Nervos-gitcoin/tree/master/task9) by @jiddukrish
* [PR](https://github.com/BCvsv/gitcoin-nervos/tree/main/Nervos%209) by @bcvsv
* [PR](https://github.com/voldown/ckb-bounty/tree/main/task9) by @voldown
* [PR](https://github.com/deryyy/gitcoin/tree/main/TASH%209) by @deryyy
* [PR](https://github.com/xinbaDev/nervos-gitcoin#gitcoin-9-initiate-withdrawal-process-from-the-layer-2-back-to-layer-1) by @xinbadev
* [PR](https://github.com/vivensy/nervos-gitcoin/tree/main/task9) by @vivensy
* [PR](https://github.com/drugurares/Gitcoin_Nervos/tree/main/task9) by @drugurares
* [PR](https://github.com/howinoc/gitcoin-nervos/tree/main/gitcoin9) by @howinoc
* [PR](https://github.com/bulentbozdag/nervos-testnet/tree/main/task-9) by @bulentbozdag
* [PR](https://github.com/icaroo/nervosTask9) by @icaroo
* [PR](https://github.com/mmcel/nervos-gitcoin/tree/main/task9) by @mmcel
* [PR](https://github.com/aliabdullah54/nervos/tree/main/task09) by @aliabdullah54
* [PR](https://github.com/puppetmaster66/nervos-hackathon/tree/master/gitcoin-9) by @puppetmaster66
* [PR](https://github.com/huazhouwang/gitcoin_ckb/tree/main/task9) by @huazhouwang
* [PR](https://github.com/hhuuaa/gitcoin_ckb/tree/main/task9) by @hhuuaa
* [PR](https://github.com/liushooter/ckb-bounty/tree/main/task10) by @liushooter
* [PR](https://github.com/nicky-ru/nervos/tree/main/gitcoin9) by @nicky-ru
* [PR](https://github.com/MASHMZC/Task---9) by @mashmzc
* [PR](https://github.com/asfman/nervos-hackathon/tree/main/task-9) by @asfman
* [PR](https://github.com/awaissaeeed/Task-9) by @awaissaeeed
* [PR](https://github.com/preet-aulokh/nervos-gitcoin/tree/main/task9) by @preet-aulokh
* [PR](https://github.com/ayush20/Nervos-Gitcoin-Hackathon/tree/main/9-initiate-withdrawal-process-from-the-layer-2-back-to-layer-1) by @ayush20
* [PR](https://github.com/mistakeone/nervos_first_try/blob/master/9%20task/README.MD) by @mistakeone
* [PR](https://github.com/homerosland/nervos-gitcoin-hack/tree/master/task9) by @homerosland
* [PR](https://github.com/berkayermis/Nervos-Gitcoin-Hackathon/tree/main/t9) by @berkayermis
* [PR](https://github.com/ysongh/Nervos-Hackathon-Submission/tree/master/task9) by @ysongh
* [PR](https://github.com/ircrp/gitcoin-nervosnetwork/tree/main/9) by @ircrp
* [PR](https://github.com/andrewpsp/gityard/tree/master/Gitcoin-9-Initiate-Withdrawal-Process-From-The-Layer-2-Back-To-Layer-1) by @andrewpsp
* [PR](https://github.com/sabaiprimo/Nervos-Hackathon/tree/main/task9) by @sabaiprimo
* [PR](https://github.com/demoncash08/Nervous-Hackathon/tree/main/task09) by @demoncash08
* [PR](https://github.com/meathewww/nervos/tree/main/t9) by @meathewww
* [PR](https://github.com/superburst/gitcoin-nervos/tree/main/t9) by @superburst
* [PR](https://github.com/stone-rocket/gitcoin_nervos/tree/main/ch9) by @stone-rocket
* [PR](https://github.com/elfelf2/gitcoin-nervos/tree/main/gitcoin9) by @elfelf2
* [PR](https://github.com/Rzbck/Rzbck-Nervos/tree/main/Gitcoin:%209-%20Initiate%20Withdrawal%20Process%20From%20The%20Layer%202%20Back%20To%20Layer%201) by @rzbck
* [PR](https://github.com/knnlrts/nervos-hackaton/tree/main/task-9) by @knnlrts
* [PR](https://github.com/Caneryy/nervos-gitcoin/tree/main/task09) by @caneryy
* [PR](https://github.com/w5pand/NervosMoon/tree/master/9) by @w5pand
* [PR](https://github.com/u2rust/NervosGitcoin/tree/main/task9) by @u2rust
* [PR](https://github.com/ufork1/NervosGitCoin/tree/master/task-9) by @ufrok1
* [PR](https://github.com/u1dxpool/nervos_gitcoin/tree/master/task9) by @u1dxpool
* [PR](https://github.com/theyashmhatre/nervos-hackathon/tree/main/9) by @theyashmhatre
* [PR](https://github.com/gabrycina/nervos/tree/main/task9) by @gabrycina
* [PR](https://github.com/deepansharya1111/Nervos-Hackathon/tree/main/Task-Gitcoin9) by @deepansharya1111
* [PR](https://github.com/scentlxss/gitcoin_nervos_tasks/tree/main/task9) by @scentlxss
* [PR](https://github.com/terryboy/nervos-gitcoin/tree/main/TASK-9) by @terryboy
* [PR](https://github.com/GentleButApril/nervos-gitcoin/tree/main/task-9) by @gentlebutapril
* [PR](https://github.com/ben-razor/nervos-hackathon/tree/main/9-withdrawal-l2-to-l1) by @ben-razor
* [PR](https://github.com/superheavytoken/Gitcoin-9-Initiate-Withdrawal-Process-From-The-Layer-2-Back-To-Layer-1) by @superheavytoken
* [PR](https://github.com/Andres-Andretti/Nervos-Hackathon/tree/main/task9) by @andres-andretti
* [PR](https://github.com/PostMan56/Nervos-BTS-Gitcoin/tree/main/Nervos-09) by @postman56
* [PR](https://github.com/ody123/nervos-grants/tree/master/Gitcoin%209)%20Initiate%20Withdrawal%20Process%20From%20The%20Layer%202%20Back%20To%20Layer%201) by @ody123
* [PR](https://github.com/Jackieyewang/Nervos-gitcoin/tree/main/task9) by @jackieyewang
* [PR](https://github.com/phungnm/nervos-gitcoin-hackathon/tree/master/task-9) by @phungnm
* [PR](https://github.com/hodlrtodlrfarmr/gitcoin_9) by @hodlrtodlrfarmr
* [PR](https://github.com/pulth/nervos-hackathon/tree/main/task9) by @pulth
* [PR](https://github.com/armujahid/nervos-hackathon/tree/master/task-9) by @armujahid
* [PR](https://github.com/simiku/nervos-gitcoin) by @simiku
* [PR](https://github.com/celoaken/gitcoin-nervos/tree/main/task9) by @celoaken
* [PR](https://github.com/jcervante/CBKTASKs/tree/main/TASK09) by @jcervante
* [PR](https://github.com/chiragbadhe/Gitcoin-Nervos/tree/main/Task9) by @chiragbadhe
* [PR](https://github.com/dikky88/CKBtest/tree/main/task9) by @dikky88
* [PR](https://github.com/niedhui/nervos-hackaton/tree/main/09) by @niedhui
* [PR](https://github.com/DragonDev1906/gitcoin-nervos/tree/main/task9) by @dragondev1906
* [PR](https://github.com/selimkafa/nervos-gitcoin/tree/main/task9) by @selimkafa
* [PR](https://github.com/bayou020/nervosnetwork/tree/main/task9) by @bayou020
* [PR](https://github.com/quocson100/task/tree/master/task9) by @quocson100
* [PR](https://github.com/sondq/nervos/tree/master/task9) by @sondq
* [PR](https://github.com/haxyz/nervos-hackathon-2021/tree/main/Withdrawal_to_Layer_1/) by @haxyz
* [PR](https://github.com/ubinix-warun/gitcoin-nervos-broaden-the-spectrum/blob/master/my-submission/task-9/README.md) by @ubinix-warun
* [PR](https://github.com/Tech-engine/nervos-network/tree/main/task9) by @tech-engine
* [PR](https://github.com/quocson95/nervos-task/tree/master/task9) by @quocson95
* [PR](https://github.com/nishuzumi/nervos-hackthon/blob/master/task9/note.md) by @nishuzumi
* [PR](https://github.com/lehoi2195/Gitcoin-Hackathon-Nervos/tree/main/task_9) by @lehoi2195
* [PR](https://github.com/duylinh196tb/Nervos-GC/blob/master/task9) by @duylinh196tb
* [PR](https://github.com/baoanh1310/nervos/tree/master/task9) by @baoanh1310
* [PR](https://github.com/kidneyweakx/nervos-gitcoin/tree/main/task-9) by @kidneyweakx
* [PR](https://github.com/long-blade/Hackathon-Nervos/tree/main/task9) by @long-blade
* [PR](https://github.com/Pfed-prog/gitcoin_nervos/blob/main/task9/) by @pfed-prog
* [PR](https://github.com/walkertraylor/gitcoin_nervos/tree/main/task9) by @walkertraylor
* [PR](https://github.com/cesheep/gitcoin-nervo/tree/main/Step%209) by @cesheep
* [PR](https://github.com/orhanors/nervos-hackhaton/tree/master/task9) by @orhanors
* [PR](https://github.com/nervos2021/gitcoin_nervos/tree/main/task9) by @nervos2021
* [PR](https://github.com/andermarce/nervos-hackaton/tree/main/task9) by @andermarce
* [PR](https://github.com/lofravaz/Nervos---Broaden-the-Spectrum#initiate-withdrawal-process-from-the-layer-2-back-to-layer-1) by @lofravaz
* [PR](https://github.com/ronalddas/gitcoin-nervos/tree/master/task9) by @ronalddas
* [PR](https://github.com/luiscrag/Gitcoin-Hackathon/tree/main/09-withdraw) by @luiscrag
* [PR](https://github.com/nhaga/nervos-hack/tree/main/task9) by @nhaga
* [PR](https://github.com/gapro8/Hackathon-Nervos/tree/master/Task-9) by @gapro8
* [PR](https://github.com/MrJacobSullivan/nervos/blob/main/submissions/task_9/TASK_9.md) by @mrjacobsullivan
* [PR](https://github.com/JEasterman/nervos/blob/master/10/README.md) by @jeasterman
* [PR](https://github.com/s1119858711/netvos-skyone/tree/main/task9) by @s1119858711
* [PR](https://github.com/mirrormirage0/nervos/tree/main/Hackathon-Task-9-Withdraw-L2-to-L1) by @mirrormirage0
* [PR](https://github.com/ririen/broaden-the-spectrum/tree/main/Initiate%20Withdrawal) by @ririen
* [PR](https://github.com/maxx6262/Gitcoin--CKB-9-Withdraw/edit/main/README.md) by @maxx6262
* [PR](https://github.com/tharunrai14/nervos-hackathon/blob/main/task09/README.md) by @tharunrai14
* [PR](https://github.com/wangjianyeART/Nervos/blob/main/task9/task9.md) by @wangjianyeart
* [PR](https://github.com/ASinghVishen/NevosBroadenTheSpectrum/tree/main/task_9) by @asinghvishen
* [PR](https://github.com/MAlvarez-0/Gitcoin/tree/main/Nervos_hackathon/Task_9) by @maxencealvarez
* [PR](https://github.com/Raiden1411/NervosHackathon_9/blob/main/README.md) by @raiden1411
* [PR](https://github.com/TTNguyenDev/Hackathon-Nervos/tree/main/task_9) by @ttnguyendev
* [PR](https://github.com/francov99/gitcoin_nervos/tree/main/Gitcoin_9) by @francov99
* [PR](https://github.com/muhdmud/gitcoinStuff/tree/main/nervosnetwork/task9) by @sidharthpunathil
* [PR](https://github.com/anhnt4288/nervous-hackathon/tree/master/task-9) by @anhnt4288
* [PR](https://github.com/yasinkirazci/nervos) by @yasinkirazci
* [PR](https://github.com/luisantoniocrag/nervos-broaden-the-spectrum/#gitcoin-9-initiate-withdrawal-process-from-the-layer-2-back-to-layer-1) by @luisantoniocrag
* [PR](https://github.com/muhdmud/gitcoinStuff/tree/main/nervosnetwork/task9) by @muhdmud
* [PR](https://github.com/el-tumero/nervos9) by @el-tumero
* [PR](https://github.com/juzkiddin/NervosHackathon/tree/main/Test9) by @juzkiddin
* [PR](https://github.com/Dex68/nervos_gitcoin/tree/main/task9) by @dex68
* [PR](https://github.com/alexchun1011/nervos/tree/master/task9) by @alexchun1011
* [PR](https://github.com/pgonday/nervosnetwork/tree/main/09.%20Withdrawal%20to%20L1) by @pgonday
* [PR](https://github.com/huutrung/gitcoin/tree/master/gitcoin-9) by @huutrung
* [PR](https://github.com/x777/Nervos-Gitconin-Hackaton) by @x777
* [PR](https://github.com/ndduc-fpt/nervos-hkt/tree/main/task9) by @ndduc-fpt
* [PR](https://github.com/happylolonly/nervos-hackathon/tree/main/task-9) by @happylolonly
* [PR](https://github.com/criusxx/nervos-gitcoin/tree/main/task9) by @criusxx
* [PR](https://github.com/johnalgo/nervoshackathon) by @johnalgo
* [PR](https://github.com/tungtobe/nervosnetwork-gitcoin/commit/dd74e5b25d19dd63e34249e0327c8f1035ceff44) by @tungtobe
* [PR](https://github.com/schen1/nvs/blob/master/task9/README.MD) by @schen1
* [PR](https://github.com/CryptoWingnut/nervosHackathon/tree/master/tasks/9) by @cryptowingnut
* [PR](https://github.com/andithemudkip/nervos/tree/main/task9) by @andithemudkip
* [PR](https://github.com/xcshuan/nervos-gitcoin/tree/main/gitcoin-9) by @xcshuan
* [PR](https://github.com/taurenshaman/Nervos202108/tree/master/projects/task-9) by @taurenshaman
* [PR](https://github.com/Domix14/Hackathon_Nervos/tree/main/9) by @domix14
* [PR](https://github.com/orangemio/nervos-grants/tree/main/task9) by @orangemio
* [PR](https://github.com/7h2x5e/nervos-gitcoin-all/tree/master/task09) by @7h2x5e
* [PR](https://github.com/hsamndo/hackathon-nervos/blob/main/README.md#gitcoin-9-initiate-withdrawal-process-from-the-layer-2-back-to-layer-1) by @hsamndo
* [PR](https://github.com/MissGeds/nervos/tree/main/task9) by @missgeds
* [PR](https://github.com/JPeterD/Gitcoin-Nervos/blob/main/Task%209/README.md) by @jpeterd
* [PR](https://github.com/L-KH/NervousHackathon/tree/main/Gitcoin9) by @l-kh
* [PR](https://github.com/assafom/hackathon-nervosnetwork/tree/main/step-9) by @assafom
* [PR](https://github.com/reza-hackathons/nervous/tree/main/t9) by @rezahsnz
* [PR](https://github.com/jinusean/nervosness/tree/main/task-9) by @jinusean
* [PR](https://github.com/rickmort/nervos-broaden-the-spectrum/tree/main/gitcoin-9) by @rickmort
* [PR](https://github.com/antonyip/nervos-gitcoin/tree/main/task-9) by @antonyip
* [PR](https://github.com/vinhbhn/nervos-gitcoin/tree/main/task-9) by @vinhbhn
* [PR](https://github.com/Soptq/2021-Nervos-Broaden-the-Spectrum-task9) by @soptq
* [PR](https://github.com/sidduHERE/Nervos/tree/main/Task9) by @sidduhere
* [PR](https://github.com/timfaner/gitcoin-task/tree/main/task9) by @timfaner

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/nervosnetwork/grants/10/100026216)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/nervosnetwork/grants/10/100026216).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.gg/gitcoin/'>Gitcoin's Discord</a>
* $2,229,860.60 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
